src/errors/test-run/templates.js

```diff
@@ -43,8 +43,9 @@ export default {
     [TEST_RUN_ERRORS.uncaughtErrorOnPage]: err => `
         A JavaScript error occurred on ${formatUrl(err.pageDestUrl)}.
         Repeat test actions in the browser and check the console for errors.
-        If you see this error, it means that the tested website caused it. You can fix it or disable tracking JavaScript errors in TestCafe. To do the latter, enable the "--skip-js-errors" option.
-        If this error does not occur, please write a new issue at:
+        This error is probably caused by the tested website and not by TestCafe.
+        To make TestCafe ignore client-side JavaScript errors, enable the "--skip-js-errors" CLI option (or the corresponding "skipJsErrors" configuration file property).
+        If this error only happens with TestCafe, please create a new issue at:
```

Comment:
+ "with TestCafe" is too imprecise
+ errors are not natural events, they can't simply 'happen'
```suggestion
        If the website only throws this error when you test it with TestCafe, please create a new issue at:
```